backends/tfhe-cuda-backend/cuda/src/integer/radix_ciphertext.cuh

```diff
@@ -207,6 +207,33 @@ __host__ void set_trivial_radix_ciphertext_async(
   }
 }
 
+// set trivial value for a single radix ciphertext
+template <typename Torus>
+__host__ void set_single_trivial_radix_ciphertext_async(
```

Comment:
Why do you define a new function? We have set_trivial_radix_ciphertext_async, couldn't you reuse it?

Reply:
`set_trivial_radix_ciphertext_async` requires array of scalars, it means one extra copy to device, all I need is single scalar value.

Reply:
done